Modules/_testcapi/exceptions.c

```diff
@@ -35,16 +48,19 @@ err_restore(PyObject *self, PyObject *args) {
     return NULL;
 }
 
-/* To test the format of exceptions as printed out. */
+/*[clinic input]
+_testcapi.exception_print
+    exception as exc: object
+    legacy: bool = False
```

Comment:
I think that `bool` is better, it used to be `"i"` legacy converted, which is just `int`.

Thanks! 👍